jobs/adept_pr/roles/autotested/files/known_failures.txt

```diff
@@ -0,0 +1,8 @@
+# <NVRA>                                   <subtest name>           <human-friendly description>
+docker-1.12.6-55.gitc4618fb.el7.x86_64     docker_cli/run_volumes   should get filtered on rhelatomic7-4-docker peons
+
+docker-1.13.1-19.git27e468e.fc26.x86_64    docker_cli/run_volumes   should get filtered on fedoratomic26-docker-latest peons
+
+docker-*                                   docker_cli/systemd_in_container should be ignored everywhere
```

Comment:
Wildcard only works for R, not V. E.g. docker-1.12.6-*. Anything like `docker-*` is probably a long-term problem and best handled in ADEPT or docker-autotest itself.

Reply:
Gotcha.  So if docker and docker-latest ever have the same version (assuming that's possible), we can just list it twice if there's an issue. Kk, np so long as that's documented.